consensus/common/src/main/java/org/hyperledger/besu/consensus/common/PoaQueryServiceImpl.java

```diff
@@ -17,17 +17,17 @@
 import org.hyperledger.besu.ethereum.chain.Blockchain;
 import org.hyperledger.besu.plugin.data.Address;
 import org.hyperledger.besu.plugin.data.BlockHeader;
-import org.hyperledger.besu.plugin.services.metrics.PoAMetricsService;
+import org.hyperledger.besu.plugin.services.metrics.PoaQueryService;
 
 import java.util.ArrayList;
 import java.util.Collection;
 
-public class PoAMetricServiceImpl implements PoAMetricsService {
+public class PoaQueryServiceImpl implements PoaQueryService {
```

Comment:
Should also implement `PoAMetricsService`.

Reply:
done